### PR TITLE
Sync install and upgrade guides with latest downstream 4.3 content

### PR DESCRIPTION
**Pull request by pmkovar**
Included updates from 3a309d9f731938ac0522089b681984f5e3c4008e.

Added more ifdef's for downstream-specific content.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @pmkovar 

This pull request needs review by: @sandrobonazzola

### CODE REVIEW

File: source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc [hunk 1]
```diff
@@ -8,12 +8,10 @@ To migrate the Data Warehouse service only, see xref:Migrating_the_Data_Warehous
 The new database server must have {enterprise-linux} 7 installed.
 Enable the required repositories on the new database server.
 
-ifdef::rhv-doc[]
 :context: migrate_DWH_DB
```
Comment (sandrobonazzola) [inline source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc:11]: Shouldn't we keep this rhv-doc only? Following link points to a file with a RHV name included

File: source/documentation/common/database/proc-Migrating_the_self-hosted_engine_database_to_a_remote_server.adoc [hunk 2]
```diff
@@ -7,10 +7,8 @@ You can migrate the `engine` database of a self-hosted engine to a remote databa
 
 The new database server must have {enterprise-linux} 7 installed and the required repositories enabled:
 
-ifdef::rhv-doc[]
 [discrete]
```
Comment (sandrobonazzola) [inline source/documentation/common/database/proc-Migrating_the_self-hosted_engine_database_to_a_remote_server.adoc:10]: isn't this rhv only related?

File: source/documentation/common/upgrade/proc-Updating_the_Red_Hat_Virtualization_Manager.adoc [hunk 3]
```diff
@@ -79,7 +87,7 @@ ifndef::SHE_minor_updates[If any kernel packages were updated, reboot the machin
 ifdef::appendices[]
 [NOTE]
 ====
-If the update from RHV 4.2.7 to RHV 4.2.8 fails with a message indicating a dependency error with an `eap7-jboss-server-migration-wildfly` package:
+If the update from {virt-product-shortname} 4.2.7 to {virt-product-shortname} 4.2.8 fails with a message indicating a dependency error with an `eap7-jboss-server-migration-wildfly` package:
```
Comment (sandrobonazzola) [inline source/documentation/common/upgrade/proc-Updating_the_Red_Hat_Virtualization_Manager.adoc:90]: above should probably be under rhv doc. In oVirt we ship wildfly, not EAP

File: source/documentation/common/upgrade/assembly-Updating_a_self-hosted_engine.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+:SHE_minor_updates:
+
+To update a self-hosted engine from your current version of 4.3 to the latest version of 4.3, you must place the environment in global maintenance mode and then follow the standard procedure for updating between minor versions.
+
+[discrete]
+include::../she/snip-Enabling_Global_Maintenance_Mode.adoc[leveloffset=+1]
+
+[discrete]
+include::proc-Updating_the_Red_Hat_Virtualization_Manager.adoc[leveloffset=+1]
+
```
Comment (sandrobonazzola) [inline source/documentation/common/upgrade/assembly-Updating_a_self-hosted_engine.adoc:14]: we should probably rename proc-Updating_the_Red_Hat_Virtualization_Manager.adoc to a more general name.

File: source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc (hunk 1 above)
Comment (pmkovar) [reply source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc:11]: Yes, we discussed renaming filenames to something less downstream-specific, but for now, we just use the RHV names. The content in `proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc` is however conditionalized.

File: source/documentation/common/database/proc-Migrating_the_self-hosted_engine_database_to_a_remote_server.adoc (hunk 2 above)
Comment (pmkovar) [reply source/documentation/common/database/proc-Migrating_the_self-hosted_engine_database_to_a_remote_server.adoc:10]: Same comment as above, I believe.

File: source/documentation/common/upgrade/assembly-Updating_a_self-hosted_engine.adoc (hunk 4 above)
Comment (pmkovar) [reply source/documentation/common/upgrade/assembly-Updating_a_self-hosted_engine.adoc:14]: Yes, agreed.

**Pull request merged**